policyengine_us/variables/gov/states/nh/tax/income/exemptions/nh_base_exemption.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class nh_base_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Hampshire base exemption household level"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NH
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        individual_income = person("dividend_income", period) + person(
```

Comment:
I don't think you need this per the above, but for future reference you can simplify this as:
```suggestion
        individual_income = add(person, period, ["dividend_income", "interest_income"])
```